dask/dataframe/groupby.py

```diff
@@ -1012,6 +1091,31 @@ def std(self, ddof=1, split_every=None, split_out=1):
         result = map_partitions(np.sqrt, v, meta=v)
         return result
 
+    @derived_from(pd.DataFrame)
+    def cov(self, ddof=1, split_every=None, split_out=1):
+        levels = _determine_levels(self.index)
+        result = aca(
+            [self.obj, self.index]
+            if not isinstance(self.index, list)
+            else [self.obj] + self.index,
+            chunk=_cov_chunk,
+            aggregate=_cov_agg,
+            combine=_cov_combine,
+            token=self._token_prefix + "cov",
+            aggregate_kwargs={"ddof": ddof, "levels": levels},
+            combine_kwargs={"levels": levels},
+            split_every=split_every,
+            split_out=split_out,
+            split_out_setup=split_out_on_index,
+        )
+
+        if isinstance(self.obj, Series):
```

Comment:
We might want to use `is_series_like` here, though it's not critical at this point

Reply:
I'm curious.  It *looks* like everything here would also work with cudf.  Is that the case?  I would be surprised if so, but pleasantly surprised :)